internal/nhooyr.io/websocket/close.go

```diff
@@ -157,7 +157,10 @@ func (c *Conn) writeClose(code StatusCode, reason string) error {
 		p, marshalErr = ce.bytes()
 	}
 
-	writeErr := c.writeControl(context.Background(), opClose, p)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	writeErr := c.writeControl(ctx, opClose, p)
```

Comment:
`c.writeControl` used a default timeout of 5 seconds, that default is only used here